base/system/services/database.c

```diff
@@ -1364,23 +1468,24 @@ ScmControlService(HANDLE hControlPipe,
                         &Overlapped);
     if (bResult == FALSE)
     {
-        DPRINT("WriteFile() returned FALSE\n");
+        DPRINT("WriteFile(%S, %d) returned FALSE\n", lpService->lpServiceName, dwControl);
 
         dwError = GetLastError();
         if (dwError == ERROR_IO_PENDING)
         {
-            DPRINT("dwError: ERROR_IO_PENDING\n");
+            DPRINT("(%S, %d) dwError: ERROR_IO_PENDING\n", lpService->lpServiceName, dwControl);
 
             dwError = WaitForSingleObject(hControlPipe,
                                           PipeTimeout);
-            DPRINT("WaitForSingleObject() returned %lu\n", dwError);
+            DPRINT("WaitForSingleObject(%S, %d) returned %lu\n", lpService->lpServiceName, dwControl, dwError);
 
             if (dwError == WAIT_TIMEOUT)
             {
+                DPRINT1("WaitForSingleObject(%S, %d) timed out\n", lpService->lpServiceName, dwControl, dwError);
```

Comment:
Shouldn't this DPRINT be also muted? It isn't a failure code path, so we don't need to always have it printed.
```suggestion
                DPRINT("WaitForSingleObject(%S, %d) timed out\n", lpService->lpServiceName, dwControl, dwError);
```
Also I think DPRINTs stuff could be probably done in a separate PR.

Reply:
**In case** this DPRINT1 would point towards a specific service taking longer than aniticipated to complete a specific action, then I would recommend to keep it a DPRINT1, as Doug did it. Because it might show us where implementations of specific services could be improved to react more quickly.

Reply:
This was the only case that there was an added DPRINT1 in PR #1225 by Giannis and I decided to leave it based on his choice there. It can be adjusted in the future as needed. Thanks.